### ISSUE TITLE
Support Angular 7

### ISSUE THREAD

**Issue by mstmvd** [Stabzs/Angular2-Toaster#181]
Please add support to angular 7.

**Comment by natcohen** [comment]
I am getting the following error with Angular 7:

```
ERROR Error: StaticInjectorError(AppModule)[ToasterContainerComponent -> ToasterService]: 
  StaticInjectorError(Platform: core)[ToasterContainerComponent -> ToasterService]: 
    NullInjectorError: No provider for ToasterService!
    at NullInjector.push../node_modules/@angular/core/fesm5/core.js.NullInjector.get (core.js:717)
    at resolveToken (core.js:954)
    at tryResolveToken (core.js:898)
    at StaticInjector.push../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (core.js:795)
    at resolveToken (core.js:954)
    at tryResolveToken (core.js:898)
    at StaticInjector.push../node_modules/@angular/core/fesm5/core.js.StaticInjector.get (core.js:795)
    at resolveNgModuleDep (core.js:17739)
    at _createClass (core.js:17786)
    at _createProviderInstance (core.js:17756)
```

Is it something that should be addressed for Angular 7 or is it an issue I am having? Did anyone experienced the same problem?

**Comment by maxan** [comment]
I'm using the Toaster with the latest version of the Angular. Have you ever faced some problem?

**Comment by Stabzs** [comment]
This is now released under version 7.0.0, although the previous version should have worked just fine.  Apologies for the delay!
